### PR TITLE
Feature/add content to single tweet

### PR DESCRIPTION
**Pull request by Tretmax**
Add render content (img and video). Marged with branch create-single-tweet

### CODE REVIEW

**Review by timurkinsan — COMMENTED** [review]
missing ";"

**Comment by alexandrtovmach** [comment]
will keep it on hold, until #7 will be merged

File: src/components/SingleTweet.tsx
Comment (VladislavPokhyl) [inline src/components/SingleTweet.tsx]: Bootstrap has own implementation of rounded images. Usage :
```
import { Image} from 'react-bootstrap'
return(
         ` <Image roundedCircle={true}/>`

)
`

File: src/components/SingleTweetComment.tsx
Comment (VladislavPokhyl) [inline src/components/SingleTweetComment.tsx]: Should've add typed props for mock/future data from DB e.g.
``` 
const SingleTweetComment = (
props:{
userNickName, 
userLogoUrl:string 
//...
}
) =>{
...
}
`

And then use in your styled components like:
```
`<NickName>@{props.userNickName}</NickName>`
`

File: package.json
Comment (ArtemLigerko) [inline package.json:27]: I think it is excessive, lets use react-bootstrap-icon only

File: src/components/SingleTweet.tsx
Comment (ArtemLigerko) [inline src/components/SingleTweet.tsx]: function is not typed

File: src/components/SingleTweet.tsx
Comment (ArtemLigerko) [inline src/components/SingleTweet.tsx]: better 'amountContent' (add 't')

**Comment by ArtemLigerko** [comment]
WM-Screenshots-20220818000737
I think it will be looking better when horizontal and vertical spaces between content will be equal

**Review by ArtemLigerko — APPROVED** [review]

**Review by alexandrtovmach — APPROVED** [review]

**Pull request merged**
